playbooks/common/openshift-cluster/upgrades/pre/verify_inventory_vars.yml

```diff
@@ -9,6 +9,19 @@
         deployment types
     when: deployment_type not in ['origin','openshift-enterprise', 'online']
 
+  # osm_cluster_network_cidr and openshift_portal_net are required when upgrading
+  # to avoid changes that may occur between releases
+  # Reference: https://bugzilla.redhat.com/show_bug.cgi?id=1451023
+  - assert:
+      that:
+      - "osm_cluster_network_cidr is defined"
+      - "openshift_portal_net is defined"
+      msg: >
+        osm_cluster_network_cidr and openshift_portal_net are required inventory variables when upgrading.
+        These variables should match what is currently used in the cluster. If you don't remember what
+        these values are you can find them in /etc/origin/master/master-config.yaml on a master with the names
+        clusterNetworkCIDR(osm_cluster_network_cidr)  and hostSubnetLength (openshift_portal_net).
```

Comment:
I'll approve it because it fixes the bug. I think it could be improved though for extra help-a-user-out value. Another task or play or something that fetched those values from a master so they didn't have to search them out themselves. I won't block on it, that might be excessive work, it's just something I thought of.